operator/src/main/resources/scripts/traceUtils.sh

```diff
@@ -104,6 +104,19 @@ function checkEnv() {
   return 0
 }
 
+#
+# getAdminUrl
+#   purpose: Get the ADMIN_URL used to connect to the admin server internally.
+#   sample:
+#     ADMIN_URL=$(getAdminUrl)
+#
+function getAdminUrl() {
```

Comment:
Would be good to document all the env vars used and even consider one for the local server as well?